R/do_adjustRtime-functions.R

```diff
@@ -582,3 +582,418 @@ adjustRtimeSubset <- function(rtraw, rtadj, subset,
     }
     rtadj
 }
+
+###########################################################
+###### LamaParama
+
+#' @title Landmark-based alignment: aligning a dataset against an external
+#' reference
+#'
+#' @aliases LamaParama-class
+#'
+#' @description
+#' Alignment is achieved using the ['adjustRtime()'] method with a `param` of
+#' class `LamaParama`. This method #' corrects retention time by aligning
+#' chromatographic data with an external reference dataset. Peaks in the
+#' experimental chromatographic data are aligned to predefined landmark
+#' features called "lamas" based on their mass-to-charge ratio (m/z) and
+#' retention time.
+#'
+#' This alignment algorithm matches chromatographic peaks from the experimental
```

Comment:
Then you could say here: *In detail, this...*

Reply:
Mmm i prefer the way you wrote it but it was very repetitive with the text below so i reorganised a bit, can you tell me if this update is okay for you ? it also reduce the text a bit (which i think is good)

Reply:
yes, the new documentation is perfect!